### ISSUE TITLE
System.Diagnostics.StackTrace.StackTraceSymbols class missing from .NET Core 2.0 

### ISSUE THREAD

**Issue by mikem8361** [dotnet/corefx#19319]
The StackTraceSymbols class in the latest .NET Core 2.0 package (I’m currently using the Microsoft.NETCore.App version 2.0.0-preview2-002093-00 package in the debuggertests repo) isn’t there.  This internal class is used by System.Private.CoreLib StackTrace code to load portable pdbs for the runtime diagnostic stack trace to get source/line info. 

If might be from this commit (hash 446e0057) by Jose on 1/20/17 to System.Diagnostics.StackTrace.csproj:
```
-  <ItemGroup Condition="'$(TargetGroup)'=='netstandard'">
+  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
    <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />
    <ProjectReference Include="..\..\System.Collections\src\System.Collections.csproj" />
    <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
    <ProjectReference Include="..\..\System.IO\src\System.IO.csproj" />
    <ProjectReference Include="..\..\System.IO.FileSystem\src\System.IO.FileSystem.csproj" />
    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
    <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj">
    </ProjectReference>
    <ProjectReference Include="..\..\System.Reflection.Metadata\src\System.Reflection.Metadata.csproj" />
    <ProjectReference Include="..\..\System.Collections.Immutable\src\System.Collections.Immutable.csproj" />
  </ItemGroup>
```

If this class/code isn’t in this assembly, the runtime won’t display any stack trace source/line info for unhandled exceptions.

The class is in the 4.4.0-beta-24913-02 package old package on myget (https://dotnet.myget.org/feed/dotnet-core/package/nuget/System.Diagnostics.StackTrace/4.4.0-beta-24913-02).

**Comment by danmosemsft** [comment]
@joperezr

**Comment by joperezr** [comment]
This has nothing to do with my change, after taking a look, this seems to be a bug with the GenFacades tool since after building the library, it has the internal type, but then we run GenFacades on it since we need to add some typeforwards to the runtime and the tool seems to be stripping this internal type out.

**Comment by joperezr** [comment]
After taking a deeper look, it's not GenFacades who is tripping it out, but the ILLinker, which runs after GenFacades on the assembly.

**Comment by lt72** [comment]
@tijoytom: assigned to you since it seems you will have a fix for this.

**Comment by mikem8361** [comment]
I'm going to do the fix for this.

**Comment by mikem8361** [comment]
Fixed PR #19364
